client/components/Item/RelatedItems.vue

```diff
@@ -109,12 +110,25 @@ export default Vue.extend({
     async refreshItems(): Promise<void> {
       this.loading = true;
 
+      let excludedArtistsIds;
```

Comment:
This can be null at the moment of performing the request

```suggestion
      let excludeArtistsIds = [];
```